docs/modules/n1ql/partials/n1ql-rest-api/admin/definitions.adoc

```diff
@@ -972,4 +973,20 @@ If specified, all completed queries with this user name are logged. +
 // end::settings[]
 
 
+[[_garbage]]
+=== Garbage
```

Comment:
This might be highly personal, but I think it would be best to refer to only the "Garbage Collector" not to "Garbage" -- typically we don't refer to unused-but-yet-to-be-freed-by-the-language-runtime memory as  "garbage".  Might look better in the documentation to only use the common language names.